src/components/Personalization/index.js

```diff
@@ -62,9 +62,12 @@ const createPersonalization = ({ config, logger }) => {
       },
       onBeforeEvent(event, options, isViewStart) {
         if (isViewStart) {
+          // If isViewStart we enable personalization
+          event.mergeQuery({ personalization: { enabled: true } });
```

Comment:
From the description in the ticket, we need to do the opposite. i.e:
```
if (!isViewStart) {
  event.mergeQuery({ personalization: { enabled: false } });
} else {
  hideContainers(prehidingId, prehidingStyle);
  event.expectResponse();
}
```

Reply:
@jonsnyder the way it is currently implemented on our edge is a little bit different. By default edge doesn't process `personalization`, you have to explicitly pass `query -> personalization -> enabled = true`, otherwise it won't work.

If we want to change the edge implementation such that personalization is always enabled and we have to explicitly disabled, then we should probably talk to @jfkhoury and Rares's team.

Let me know your thoughts.

Reply:
I think all the other queries are `enabled: true` by default, so it's a little sad if personalization is inconsistent. That doesn't mean it's the wrong thing to do, but we should probably scrutinize it a bit more. I do think we should pass this one by Rares's team.

Reply:
@Aaronius and @jonsnyder I have checked our edge code, by default all features are enabled, so I think it makes sense to be consistent.

I'll adjust the code accordingly and I'll open a PR for our backend code to change the way we handle personalization.